internal/usergroup/const_windows.go

```diff
@@ -0,0 +1,10 @@
+package usergroup
+
+const (
+	SeTakeOwnershipPrivilege = "SeTakeOwnershipPrivilege"
+)
+
+const (
+	ContainerAdministratorSidString = "S-1-5-93-2-1"
+	ContainerUserSidString          = "S-1-5-93-2-2"
+)
```

Comment:
I'm wondering (but would have to get more familiar with these), if these should remain publicly available somewhere if these are magic values that may be relevant outside of our code. If that's the case, we really need to document them though (and even consider making then non-platform-specific).